src/api/grainConfig.test.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+import {parser, toDistributionPolicy, type GrainConfig} from "./grainConfig";
```

Comment:
nit: we usually have a blankline after the flow pragma before the first import

Reply:
One nit a day makes the codebase go hooray